src/model.jl

```diff
@@ -20,7 +18,11 @@ function fit_profile(profile_values::Vector{Float64},
     @variable(model, x >= 0)
 
     # Create the objective function
-    @NLobjective(model, Min, (sum((profile_values[i])^x for i in 1:total_hours) - target_mean*total_hours)^2)
+    @NLobjective(
+        model,
+        Min,
+        (sum((profile_values[i])^x for i = 1:total_hours) - target_mean * total_hours)^2
```

Comment:
It would be useful to write the math for this in the docs. Something like

```math
\min \qquad \bigg(\mu h - \sum_{i = 1}^h p_i^x\bigg)^2
```

It would also make it easier to read with less parenthesis, that's why I moved things around.
I had the impression the problem was

```math
\min \qquad \sum_{i=1}^h (\mu - p_i^x)^2,
```

i.e., you want each `p_i^x` to be close to `\mu`.

Reply:
Never mind my comment, I read that you are just worried about the mean of the new `p_i^x`, not the individual `p_i^x` value, so the first definition is correct. I would maybe write as

```math
\min \qquad \frac{1}{2}\bigg(\mu - \frac{1}{h}\sum_{i=1}^h p_i^x\bigg)^2
```

Reply:
I realized something about the problem. You are doing nonlinear least squares of a single variable with a single equation. So it makes sense to ask whether the equation already has a solution. Under conditions on the profile values and the target_mean, it has, and in that case it would be simpler to use a root-finding algorithm.
For instance:

```julia
using Roots
profile_values = [0.2, 0.5, 0.9, 1.0, 0.4, 0.1, 0.0] 
target_mean = 0.5
find_zero(
	x -> sum(profile_values .^ x .- target_mean),
	[1e-2, 1.0],
)
```

Depending on what you want, and the assumptions that you can make, this can be used.